bitmind/synthetic_image_generation/utils/image_utils.py

```diff
@@ -1,4 +1,7 @@
 import PIL
+import os
```

Comment:
[Suggestion] adding this to the list of things that may belong in a separate repo

Reply:
This one is necessary for image annotation generation, but we may want to reorganize all the utils in the future.